README.md

```diff
@@ -5,6 +5,21 @@ This is a very small tool to help you burn **mycobot** Firmware, it's not as big
 - Support local firmware
 - Support remote firmware,
 
+### Permissions
+
+If you need to communicate with your device, please ensure that you have the
```

Comment:
Working: Would be better to ref. Arduino docs: https://www.arduino.cc/en/guide/linux#toc6

example: https://github.com/eacousineau/repro/blob/f48df63/robots/mycobot_setup/README.md#permissions